pkgs/by-name/gz/gz-math/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  cli11,
+  cmake,
+  eigen,
+  fetchFromGitHub,
+  gz-cmake,
+  gz-utils,
+  gtest,
+  python3,
+  ruby,
+  swig,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-math";
+  version = "8.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazebosim";
+    repo = "gz-math";
+    tag = "gz-math${lib.versions.major finalAttrs.version}_${finalAttrs.version}";
+    hash = "sha256-E7u3EtpqNLvcqI5ycezwwAlbVHM3JdqeyLFWYlEaOYo=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gz-cmake
+    gz-utils
+    python3
+  ];
+
+  BuildInputs = [
+    cli11
+    eigen
+    ruby
+    swig
+  ];
+
+  doCheck = true;
+
+  nativeCheckInputs = [ python3 ];
+
+  checkInputs = [ gtest ];
+
+  cmakeFlags = [
+    (lib.cmakeBool "GZ_UTILS_VENDOR_CLI11" false)
```

Comment:
GitHub search does not find this flag within the repository; Is CLI11 used by this project?